contracts/infinity/diamond/AppStorage.sol

```diff
@@ -52,6 +52,7 @@ struct AppStorage {
     mapping(address => Account.State) a; // Mapping of a user address to account state.
     uint256 lastERC721Received; // Last ERC721 Toucan Retirement Certificate received.
     uint256 fee; // Aggregator fee charged on all retirements to 3 decimals. 1000 = 1%
+    uint256 daoFeeShare; // Share of 'fee' for the DAO wallet, per KIP-33. 3 decimals. 1000 = 1%
```

Comment:
Should we add a `feeDivisor` field and remove the magic numbers from the libraries?

Reply:
Yeah, definitely, I was thinking that as well as I was typing it in. I'll add it.